### ISSUE TITLE
`v-slot:event="{ event, cell }`

### ISSUE THREAD

**Issue by fratzinger** [antoniandre/vue-cal-v4#383]
Hey Antoni,

I need the ability to format the date based on the cell in a custom `#event` slot.

If I'm correct, it's not possible to achieve following in a custom `#event` slot:
```pug
  small.days-to-end(v-if="event.daysCount > 1 && (event.segments[cell.formattedDate] || {}).isFirstDay")
    | &nbsp;+{{ event.daysCount - 1 }}{{ (texts.day[0] || '').toLowerCase() }}
```

What about passing the cell to the slot, to make this possible?

Love your work!

**Comment by antoniandre** [comment]
Hi @fratzinger, 
Sorry for the delay and thanks for the feedback! 

If you could give me your use-case (ideally a reproduction link like a Codepen, you can fork this one for instance https://codepen.io/antoniandre/pen/xxZbKrO) and what exactly you want to do, I might be able to suggest an alternative, or update the codebase. :)

**Comment by Levonlk** [comment]
Hi @antoniandre 
@fratzinger probably means that it is currently not possible to use a custom event to keep the time format for multiple day events and not display "+ 1d." in the last cell (last day of multiple days event). For this reason, a "cell" is required in the slot. I also got this problem now.
Thanks for your great and cool job!

**Comment by frankkoornstra** [comment]
This would be a very welcome addition! Having that same problem atm: I'm custom rendering events but want to show the date differently depending on the cell:
* First day: `22:00` (start time)
* Second day: `cont`
* Third day: `11:00` (end time)

Passing the cell (or even the date of the cell in which the event is rendered) to the slot would solve this problem.

**Comment by StubyProjects** [comment]
@frankkoornstra Yes this is exactly what we are looking for too. We have multiday events and don't want to display the start date of the first day on day two. So we need the cell in the #event slot

**Comment by frankkoornstra** [comment]
@antoniandre any chance of this happening? I've looked into the code but it's too complicated for me to open a PR, sorry! If you can drop a hint on how to solve it, maybe I can find a way to do it or find someone that can help.

Also, would you mind signing up this repo to [Bountysource](https://bountysource.com/)? I'm definitely willing to put up some money for this to get solved.
